sklearn/metrics/scorer.py

```diff
@@ -487,6 +489,9 @@ def make_scorer(score_func, greater_is_better=True, needs_proba=False,
 mean_absolute_error_scorer = make_scorer(mean_absolute_error,
                                          greater_is_better=False)
 mean_absolute_error_scorer._deprecation_msg = deprecation_msg
+neg_mape_scorer = make_scorer(mean_absolute_percentage_error,
+                              greater_is_better=False)
+
```

Comment:
Maybe remove the new line here. When there is no clear rule, my advice would be to follow the same implicit convention as the code you are changing.